deno/deepgram-language-detection/README.md

```diff
@@ -0,0 +1,66 @@
+# ⚡ deepgramLanguageDetection()
+
+A Deno Cloud Function for detecting language from audio using [Deepgram](https://deepgram.com/)
+
+_Example function payload:_
+
+```json
+{
+  "fileUrl": "https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Successful function response::_
+
+```json
+{ "deepgramData": {}, "success": true }
+```
+
+_Error function response:_
+
+```json
+{ "message": "error message", "success": false }
+```
+
+## 📝 Variables
+
+List of variables used by this cloud function:
+
+**DEEPGRAM_API_KEY** - Your Deepgram API key.
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd deno/deepgram-transcribe-video
```

Comment:
Please fix the instruction here

Reply:
It is not a familiar error!
 let me check once more

Reply:
@gewenyu99 
As you know
This command is to clone `open-runtimes/examples` not my fork ( It will only work after it is merged!!
So there is no deno/

I included this so that after the pull request was merged, maintainers wouldn't need to edit it. 

For now, clone this https://github.com/Rinshin-Jalal/open-runtimes-examples/
and change the branch to `deepgram-language-detection-deno`

Reply:
Yes @Rinshin-Jalal, as @stnguyen90 mentioned, the instructions still do not match the actual folder name.

Reply:
@gewenyu99, can this be resolved now?